packages/server-settings/src/components/ServerSettingsView/index.tsx

```diff
@@ -130,7 +130,8 @@ export const ServerSettingsView: React.FC<Props> = (props: Props) => {
     SECURITY_AUTHENTICATE_ENDPOINT,
     () => new OpenSRPService(SECURITY_AUTHENTICATE_ENDPOINT, baseURL).list(),
     {
-      onError: () => sendErrorNotification(t('An error occurred')),
+      onError: () =>
+        sendErrorNotification(t('There was a problem authenticating User Location settings')),
```

Comment:
This endpoint would return the user assignment data, i.e. linked users, location, teams , etc.